app/overnight/expired-insurance.js

```diff
@@ -16,24 +16,38 @@ const setExpiredInsuranceToBreach = async (today, user, t) => {
           [Op.lt]: today
         }
       },
-      include: [{
-        model: sequelize.models.dog,
-        as: 'dog',
-        include: [{
-          model: sequelize.models.status,
-          as: 'status'
-        },
+      include: [
         {
-          model: sequelize.models.insurance,
-          as: 'insurance'
-        }]
-      }],
+          model: sequelize.models.dog,
+          as: 'dog',
+          include: [
+            {
+              model: sequelize.models.status,
+              as: 'status'
+            },
+            {
+              model: sequelize.models.insurance,
+              as: 'insurance'
+            },
+            {
+              model: sequelize.models.dog_breach,
+              as: 'dog_breaches',
+              include: [
+                {
+                  model: sequelize.models.breach_category,
+                  as: 'breach_category'
+                }
+              ]
+            }
+          ]
+        }
+      ],
       transaction: t
     })
+    const dogService = ServiceProvider.getDogService()
 
     for (const toUpdate of setToBreach) {
-      console.log(`Updating dog ${toUpdate.dog.index_number} to In breach`)
-      await updateStatusOnly(toUpdate.dog, statuses.InBreach, user, t)
+      await dogService.setBreach(toUpdate.dog, [breachReasons.NOT_COVERED_BY_INSURANCE], user, t)
```

Comment:
Surely this should be set to INSURANCE_EXPIRED

Reply:
very true 👍 good spot, thanks